contracts/WalletSimple.sol

```diff
@@ -340,10 +339,6 @@ contract WalletSimple is ReentrancyGuard, IERC721Receiver, ERC1155Receiver {
     address tokenContractAddress
   ) external onlySigner {
     IForwarder forwarder = IForwarder(forwarderAddress);
-    require(
```

Comment:
Are we removing this because we know that the Forwarder is our contract and don't need this check?

Reply:
Correct and it saves gas